{{cookiecutter.github_repository}}/docs/graphql/0-overview.md

```diff
@@ -0,0 +1,18 @@
+# Graphql API
+
+## Endpoint
+```
+/graphql
```

Comment:
```suggestion
POST /graphql
```

Reply:
We can also add a sample query formation here:

__NOTE__: Payload can be any query/mutation in GraphQL sent as a HTTP request

```json
{
    "query": "query {
            me {
                id
                firstName
                lastName
                email
            }
        }",
   "operationName": "GetMyInfo",
   "variables": {}
}
```